container_registry/container_analysis/poll_discovery_finished.go

```diff
@@ -51,9 +51,19 @@ func pollDiscoveryOccurrenceFinished(resourceURL, projectID string, timeout time
 			return nil, fmt.Errorf("timeout while retrieving discovery occurrence")
 		case <-ticker.C:
 			req := &grafeaspb.ListOccurrencesRequest{
+				Parent: fmt.Sprintf("projects/%s", projectID),
+				// Vulbnerability discovery occurrences are always associated with the
```

Comment:
Sorry I missed this, typo in vulnerability.

Reply:
good catch